.claude/commands/cleanup-builds.md

```
@@ -0,0 +1,29 @@
+# Cleanup Dev Builds
+
+Reclaim disk space taken by tagged dev artifacts produced by `./scripts/reload.sh --tag <tag>`. Each tagged build is multi-GB of DerivedData plus per-tag sockets and logs.
+
+## Steps
+
+1. **Preview first.**
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)